decoder/vaapidecoder_h264.cpp

```diff
@@ -1819,14 +1819,24 @@ YamiStatus VaapiDecoderH264::decodeNalu(NalUnit* nalu)
     return status;
 }
 
+void VaapiDecoderH264::flush(void)
+{
+    decodeCurrent();
+    m_dpb.flush();
+    m_newStream = true;
+    m_endOfStream = false;
+    m_endOfSequence = false;
+    m_currPic.reset();
+    m_prevPic.reset();
+    m_currSurface.reset();
```

Comment:
But flush is not stop, it do not need deinit Va. Maybe we can put all context reset thing to a standalone function like resetDecoderContext.